### PR TITLE
Remove Save button and functionality from Settings

### PR DESCRIPTION
**Pull request by joshuatbrown**
## Issues covered
#1243

## Description
Removed the Save button and importKey from Settings. Switched SecureField to a plain old Text with dots so the user can't edit their private key. Also added an accessibility label to that Text since otherwise it's just a bunch of dots, and I guess the user might figure out what that is, but a label telling them seems more appropriate.

## How to test
1. Navigate to Settings
2. Observe Private Key section

## Screenshots/Video
| Updated Settings | With VoiceOver label (bottom) |
|--------|--------|
| IMG_44058AFF4A45-1 | IMG_0078 |

### CODE REVIEW

File: Nos/Views/SettingsView.swift [hunk 1]
```diff
@@ -51,27 +44,13 @@ struct SettingsView: View {
         Form {
             Section {
                 HStack {
-                    SecureField(String(localized: .localizable.privateKeyPlaceholder), text: $privateKeyString)
+                    Text("•••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••")
```
Comment (joshuatbrown) [inline Nos/Views/SettingsView.swift:47]: 63 dots since that's what the old one showed. 🤷

Comment (martindsq) [reply Nos/Views/SettingsView.swift:47]: I see what you are doing. It makes no sense to use a secure field if we always show the same thing.
However, you can use `String(repeating: "•", count: 63)` instead.

**Review by martindsq — CHANGES_REQUESTED** [review]
I still have no clue what the Save button did xd
I just left a minor comment I think you will like.

**Comment by joshuatbrown** [comment]
@martindsq awesome suggestion! Done!

**Review by martindsq — APPROVED** [review]
Excellent, let's ship it!

**Pull request merged**